### DISCUSSION TITLE
DDR5 model

### DISCUSSION THREAD

**Discussion by kosmalex** [gem5/gem5#1719] [Q&A] [answered]
Why are there missing default parameters in the DDR5 model of GEM5? For example parameters tCS or tREFI do not have a default value set. The file containing DDR models is: <path-to-gem5>/src/mem/DRAMInterface.py

**Comment by Will-Shaddix** [comment, marked as answer]
The DDR5_4400_4x8 model does have defaults for these values. The other DDR5 models inherit DDR5_4400_4x8 so if they don't specify a different value, they'll have the value listed in DDR5_4400_4x8.

**Reply by kosmalex** [reply]
Thanks a lot, I was working with an older containerized version of gem5, and didn't see the other ddr5 models in this repo XD.
